app/views/safetyDiagnosticTool/versions/v1/layout.html

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
A layout for each page seems like a lot. I see that this is so you can re-use the changLog contents. Maybe an alternative could be a "changelog" component that accepts the version number and content location?